src/audio/mod.rs

```diff
@@ -185,7 +185,7 @@ impl Stream {
             // Always consume the frame from PulseAudio. That way this thread doesn't end up with
             // huge jumps or large number of dropped messages.
             if let Err(e) = self.read_frame() {
-                log::error!("failed to read audio frame: {}", e);
+                log::error!("failed to read audio frame: {:#}", e);
```

Comment:
Is `{:#}` a thing? Might need to be `{:#?}`
https://doc.rust-lang.org/std/fmt/#sign0

Reply:
Disregard me: https://docs.rs/anyhow/1.0.38/anyhow/struct.Error.html#display-representations
TIL